src/pages/experimental/edit-pattern/index.mdx

```diff
@@ -0,0 +1,197 @@
+
+Warn the user when the consequence of editing could damage resources or connections in their system.
+
+</PageDescription>
+
+#### Status:
+
+[Experimental](/experimental/overview/)
+
+#### Maintainers:
+
+[Vikki Paterson](https://github.com/vikkipaterson), [Chloe Poulter](https://github.com/chloepoulterdesign)
+
+<AnchorLinks>
+
+  <AnchorLink>Low-impact edit</AnchorLink>
+  <AnchorLink>High-impact edit</AnchorLink>
+  <AnchorLink>Optional extras</AnchorLink>
+
+</AnchorLinks>
+
+![Contextual image of an edit modal](images/01.png)
+
+<Caption>Example of an edit modal in context</Caption>
+
+## Low-impact edit
+
+Allow editing with no warning. Typically used when editing doesn’t have a wider impact.
+
+<Row>
+<Column colLg={8}>
+
+![Example of an edit button in a table](images/02.png)
+
+<Caption>Example of an edit button in a table</Caption>
+
+</Column>
+</Row>
+
+### Modal
+
+If an object was created in a modal, its editable fields can be made available in a modal. A modal can also be used where a small subset of fields are editable.
+If enough fields are editable to require scrolling, use a side-panel or full-page edit dialog instead.
+
+<Row>
+<Column colLg={8}>
+
+![Example of a low consequence edit action in a modal](images/03.png)
+
+<Caption>Example of a low conssequence edit action in a modal</Caption>
+
+</Column>
+</Row>
+
+### Side-panel
+
+If an object was created in a side panel, its editable fields can be made available in a side-panel. A side-panel can also be used where context in the main view of the screen is useful in the editing flow.
+
+<Row>
+<Column colLg={8}>
+
+![Example of a low impact edit action in a side panel](images/04.png)
+
+<Caption>Example of a low impact edit action in a side panel</Caption>
+
+</Column>
+</Row>
+
+## High-impact edit
+
+Warn the user of the consequences of editing. Typically used when editing has a wider impact.
+Inform users of the likely consequences of the edit to their system, and inform them that editing cannot be undone. Include a warning stage.
+
+### Modal
+
+A high impact edit action of only a few fields can use a Danger modal. A second confirm edit modal should also be inserted before the changes are saved. If desired, the confirmation modal can include a [summary of changes](#summary-of-changes).
+
+<Row>
+<Column colLg={8}>
+
+![Example of a high-impact editing modal and consequent warning modal](images/05.png)
+
+<Caption>
+  Example of a high-impact editing modal and consequent warning modal
+</Caption>
+
+</Column>
+</Row>
+
+### Side-panel
+
+If an object was created in a side panel, or there are too many editable fields for a modal, a slide-over panel can be used.
+
+<Row>
+<Column colLg={8}>
+
+![Example of a high impact edit action in a side-panel](images/06.png)
+
+<Caption>Example of a high impact edit action in a side-panel</Caption>
```

Comment:
```suggestion
<Caption>Example of a high-impact edit action in a side-panel</Caption>
```

Reply:
I thought you'd inserted the hyphen into `side panel` in other reviews, but could be entirely mistaken